.gitmodules

```diff
@@ -244,3 +243,28 @@
 	branch = branch-0.0.1
 	shallow = true
 	fetchRecurseSubmodules = false
+#sky130_fd_pr_reram
```

Comment:
Your missing the `latest` submodule;
```
[submodule "libraries/sky130_fd_io/latest"]
	path = libraries/sky130_fd_io/latest
	url = https://github.com/google/skywater-pdk-libs-sky130_fd_io.git
	branch = main
	shallow = true
	fetchRecurseSubmodules = true
```

Can you make sure the new modules are also sorted correctly? (newest -> oldest)